discord/ext/commands/core.py

```diff
@@ -829,6 +835,10 @@ async def prepare(self, ctx: Context) -> None:
                 await self._max_concurrency.release(ctx)  # type: ignore
             raise
 
+    @property
+    def cooldown(self) -> Optional[Cooldown]:
+        return self._buckets._cooldown
```

Comment:
Couldn't this property just be documented here instead of in the main docstring? Just to be consistent with rest of the code's document style?

Reply:
Properties of the `Command` class uses this style, as seen in `Command.callback` etc.

Reply:
Right, LGTM.